homeassistant/components/ecobee/humidifier.py

```diff
@@ -43,6 +43,32 @@ def __init__(self, data, thermostat_index):
 
         self.update_without_throttle = False
 
+    @property
+    def name(self):
+        """Return the name of the humidifier."""
+        return self._name
+
+    @property
+    def unique_id(self):
+        """Return unique_id for humidifier."""
+        return f"{self.thermostat['identifier']}"
+
+    @property
+    def device_info(self):
+        """Return device information for the ecobee humidifier."""
+        try:
+            model = f"{ECOBEE_MODEL_TO_NAME[self.thermostat['modelNumber']]} Thermostat"
+        except KeyError:  # pragma: no cover
```

Comment:
It's better to exclude the whole module from coverage calculation. Please add it in `.coveragerc`.

Reply:
Ok. Updated. Thanks.